app/models/user.rb

```diff
@@ -2,8 +2,9 @@
 # Fields are all devise settings; most of the methods relate to call list mgmt.
 class User
   include Mongoid::Document
-  include Mongoid::Enum
   include Mongoid::Userstamp::User
+  extend Enumerize
+  extend ActiveModel::Naming
```

Comment:
I have the same flaw, when you point something out there are usually four fingers pointing back at you. Habit that I'm breaking over time. You in no sense want to end up with superfluous commits but for feedback this stuff can be pretty valuable (if for example you bundle like ten fixes into one that says "Feedback", it's perfectly reasonable to just use that commit hash for all of them).